prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/FHIRBundleHelpersTests.kt

```diff
@@ -160,7 +183,7 @@ class FHIRBundleHelpersTests {
         val provenance = bundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
         val outs = provenance.target
         val receiversOut = outs.map { (it.resource as Endpoint).identifier[0].value }
-        assert(receiversOut.isEmpty())
+        assertThat(receiversOut.isEmpty())
     }
```

Comment:
Move the isEmpty() outside of the assertThat, do the same for the others.

Reply:
Assertions updated.